compilers/esm.js

```diff
@@ -101,7 +101,7 @@ exports.compile = function(load, opts, loader) {
       lowResolutionSourceMap: opts.lowResSourceMaps
     });
 
-    var tree = load.metadata.parseTree || compiler.parse(load.source, load.path);
+    var tree = load.metadata.parseTree || compiler.parse(source, load.path);
```

Comment:
oh, dang, you're right

problem is I work on the systemjs-builder in my node_modules folder then cloned this repo and haphazardly copied the changes onto it